src/Analyzer/Pass/ConfigurablePassInterface.php

```diff
@@ -5,12 +5,15 @@
 
 namespace PHPSA\Analyzer\Pass;
 
-use Symfony\Component\Config\Definition\Builder\TreeBuilder;
-
 interface ConfigurablePassInterface
 {
     /**
-     * @return TreeBuilder
+     * @return \Symfony\Component\Config\Definition\Builder\NodeDefinition
+     */
+    public static function getConfiguration();
+
+    /**
+     * @return string
      */
-    public function getConfiguration();
+    public static function getName();
```

Comment:
I had to define it as a static method as I need to get an analyzer's name before they are instantiated (see the Factory::getExpressionPasses method)